tests/unit/source_code/test_lsp_plugin.py

```diff
@@ -48,3 +74,112 @@ def test_pylsp_lint(workspace):
             'tags': [],
         },
     ]
+
+
+def test_pylsp_lint_no_dbr_version(workspace, config):
+    code = 'sc.emptyRDD()\ndf.groupby("id").applyInPandas(udf)'
+    _, doc = temp_document(code, workspace)
+    config.update(
+        {
+            'plugins': {
+                'pylsp_ucx': {'enabled': True, 'isServerless': False, 'dataSecurityMode': 'USER_ISOLATION'},
+            }
+        }
+    )
+
+    diagnostics = sorted(lsp_plugin.pylsp_lint(config, doc), key=lambda d: d['code'])
+    assert diagnostics == [
+        {
+            'range': {'start': {'line': 0, 'character': 0}, 'end': {'line': 0, 'character': 11}},
+            'code': 'legacy-context-in-shared-clusters',
+            'source': 'databricks.labs.ucx',
+            'message': 'sc is not supported on UC Shared Clusters. Rewrite it using spark',
+            'severity': 1,
+            'tags': [],
+        },
+        {
+            'range': {'start': {'line': 0, 'character': 0}, 'end': {'line': 0, 'character': 13}},
+            'code': 'rdd-in-shared-clusters',
+            'source': 'databricks.labs.ucx',
+            'message': 'RDD APIs are not supported on UC Shared Clusters. Rewrite it using DataFrame API',
+            'severity': 1,
+            'tags': [],
+        },
+    ]
+
+
+def test_pylsp_no_config(workspace, config):
+    code = 'sc.emptyRDD()'
+    _, doc = temp_document(code, workspace)
+    diagnostics = sorted(lsp_plugin.pylsp_lint(config, doc), key=lambda d: d['code'])
+    assert diagnostics == []
```

Comment:
Indepedent of the code, should this always return no diagnostic if the config is not set?

Reply:
i think so